src/installation/guides/raspberry-pi.md

```diff
@@ -0,0 +1,201 @@
+$ parted mktable msdos
+$ parted mkpart primary fat32 2048s 256MB
+$ parted toggle 1 boot
+```
+
+Create the rootfs partition, extending to the end of the device:
+
+```
+$ parted mkpart primary ext4 256MB -1
+$ parted quit
+```
+
+Create the filesystems in the SD card:
+
+```
+$ mkfs.vfat /dev/mmcblk0p1 # change this to match your SD card and FAT32 partition
+$ mkfs.ext4 -O '^has_journal' /dev/mmcblk0p2 # change this to match your SD card and ext4 partition
+```
+
+The `-O ^has_journal` option disables journaling on ext4 partition, which will
+extend the life of your drive (usually Flash drives).
+
+### Preparing target rootfs directory
+
+```
+$ mkdir rootfs
+# mount /dev/mmcblk0p2 rootfs/
+# mkdir rootfs/boot
+# mount /dev/mmcblk0p1 rootfs/boot
+```
+
+Unpack the tarball into the target rootfs directory and sync to make sure files
+are written to storage:
+
+```
+# tar xvfJp void-rpi*-PLATFORMFS-%DATE.tar.xz -C rootfs
+# sync
+```
+
+Do not forget to unpack the rootfs as `root`, and with the `-p` flag, to set
+appropriate permissions.
+
+The `/boot` partition must be added to `/etc/fstab`:
+
+```
+# echo '/dev/mmcblk0p1 /boot vfat defaults 0 0' >> rootfs/etc/fstab
+```
+
+Umount the SD card filesystems from target rootfs directory.
+
+You can tweak kernel boot cmdline arguments in the `rootfs/boot/cmdline.txt`
+file.
+
+Insert the SD card and test that the Raspberry PI boots correctly.
+
+## First Boot
+
+The `root` password is `voidlinux`.
+
+### Set the system time
+
+Before it is possible to install or upgrade packages, it is necessary to set the
+clock, or you will see HTTPS certificate errors. The Raspberry Pi does not have
+a battery-backed clock, so you must set the time manually.
+
+Install and enable [an NTP server](../../config/date-time.md#ntp) and wait for
+the date and time to be set.
+
+??? However, if you still encounter SSL related errors or the time is still not
+correct, it might be worth to reconfigure package chrony (`xbps-reconfigure -f
+chrony`) because some images tend to not create chrony user and group, often
+required by ntpd. ???
+
+### Enabling hardware RNG device
+
+By default the
+[HWRNG](https://en.wikipedia.org/wiki/Hardware_random_number_generator) device
+is not usedm which may result in the random devices taking long to seed on boot
+(annoying if you want to start `sshd` and expect to be able to connect
+immediately).
+
+First, add the RPi's HWRNG kernel module (`bcm2835_rng`) to a file in
+`/etc/modules-load.d`, such as `/etc/modules-load.d/rng.conf`.
+
+Then install `rng-tools` and
+[enable](../../config/services/index.md#enabling-services) the `rngd` service so
+that the kernel's randomness will be seeded by the `hwrng` device.
+
+## Applications
+
+### X11
+
+At a minimum, you will need to install `xorg-minimal` and `xf86-video-fbturbo`.
+Client applications are available in `xorg-apps`, and X fonts are available in
+`xorg-fonts`.
+
+Make sure the user running X is part of the groups `audio` and `video`.
```

Comment:
I would redirect to the Xorg page and just say that `xf86-video-fpturbo` is necessary. Not sure it is, though, since we have mode setting working in RPis now.

Reply:
Done, although i've left the "Make sure the user ..." line in; let me know if that can be removed.